### ISSUE TITLE
About the valiadation metrics

### ISSUE THREAD

**Issue by Longzhong-Lin** [sshaoshuai/MTR#56]
![image](https://github.com/sshaoshuai/MTR/assets/81036197/24de6855-3544-4e18-afd8-885153a1a775)
Firstly much thanks for opensourcing such a wonderful work!

 I saw in your code there is a method to compute the Waymo Motion metrics locally. So I wonder where do the numbers of `Performance on the validation set` in README come from, local computation or official eval server?

Furthermore, before submitting to test leaderboard, did you finetune the model on data including validation set? It would be very kind of you to explain more details about the finetuning.

**Comment by sshaoshuai** [comment]
The provided numbers are evaluated with our local evaluation codes, which are almost exactly the same as the official eval server (the mAP may be slightly different (< 0.5% I think) and I do not know why).

**Comment by Longzhong-Lin** [comment]
Got it! Much thanks for such a useful & in-time reply!
